lib/api.ts

```diff
@@ -51,12 +55,40 @@ export const getSingleCategoryPost = (
 
   // If file exists, it is not an index page.
   if (fs.existsSync(filePath)) {
-    return { isIndex: false, ...matter(fs.readFileSync(filePath, 'utf8')) };
+    return { isIndex: false, ...readFile(filePath) };
   }
 
   if (fs.existsSync(indexPath)) {
-    return { isIndex: true, ...matter(fs.readFileSync(indexPath, 'utf8')) };
+    return { isIndex: true, ...readFile(indexPath) };
   }
 
   return;
 };
+
+/**
+ * Retrieves navigation data from /content/docs
+ */
+export const getSupportNavigation = () => {
```

Comment:
This only supports 1 level of nav nesting

Reply:
That should be fine since the content reflects that need for now. If we get a feature request to go 2 level deep, we'll address it then.